jiant/tasks/lib/pawsx.py

```diff
@@ -96,13 +96,11 @@ def get_test_examples(self):
     def _create_examples(cls, lines, set_type):
         examples = []
         for (i, line) in enumerate(lines):
+            if i == 0:
+                continue
```

Comment:
Can you add a comment explaining this?

Reply:
Added.

Reply:
Mind adding it again?

Reply:
Whoops, forgot to push it.

Reply:
Can you explain why it is necessary to skip the first line? Is it a header?

Reply:
Done.